package.json

```diff
@@ -91,6 +91,8 @@
     "@storybook/react-webpack5": "^7.1.0-alpha.1",
     "@storybook/theming": "^7.1.0-alpha.1",
     "@testing-library/jest-dom": "^5.10.1",
+    "@testing-library/preact": "^3.2.3",
```

Comment:
It looks like we don't use this. Is that true?

Reply:
i tried running `yarn remove @testing-library/preact` on it but it ended up breaking functionality for the preact-hooks library. not sure if i did the removal wrong.